### PR TITLE
Add new getter of API v1.23

### PR DESCRIPTION
**Pull request by yuting-liu**
v1.23 API changes including following:
* `GET /containers/json` returns the `state` of the container, one of `created`, `restarting`, `running`, `paused`, `exited` or `dead`.
* `GET /containers/json` returns the mount points for the container.
* `GET /networks/(name)` now returns an `Internal` field showing whether the network is internal or not.
* `GET /networks/(name)` now returns an `EnableIPv6` field showing whether the network has ipv6 enabled or not.

---
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/956)

### CODE REVIEW

File: src/main/java/com/github/dockerjava/api/model/Container.java [hunk 1]
```diff
@@ -83,6 +89,12 @@
     @JsonProperty("NetworkSettings")
     private ContainerNetworkSettings networkSettings;
 
+    /**
+     * @since ~{@link RemoteApiVersion#VERSION_1_23}
+     */
+    @JsonProperty("Mounts")
+    private ContainerMount[] mounts;
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/model/Container.java]: Please use List or Set

File: src/main/java/com/github/dockerjava/api/model/ContainerMount.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    @Override
+    public boolean equals(Object o) {
+        return EqualsBuilder.reflectionEquals(this, o);
+    }
+
+    @Override
+    public int hashCode() {
+        return HashCodeBuilder.reflectionHashCode(this);
+    }
+}
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/model/ContainerMount.java]: add new line

File: src/main/java/com/github/dockerjava/api/model/Container.java (hunk 1 above)
Comment (yuting-liu) [reply src/main/java/com/github/dockerjava/api/model/Container.java]: Sure. I could change this.
BTW, for `public ContainerPort[] ports`, do we also want to use List or Set for it?

Comment (KostyaSha) [reply src/main/java/com/github/dockerjava/api/model/Container.java]: old code not changed, will be changed in future

**Pull request merged**